tesseract/tesseract_planning/tesseract_process_managers/src/process_managers/freespace_process_manager.cpp

```diff
@@ -46,10 +46,12 @@ bool FreespaceProcessManager::init(ProcessInput input)
 
   // Create the dependency graph
   if (console_bridge::getLogLevel() == console_bridge::LogLevel::CONSOLE_BRIDGE_LOG_DEBUG)
-    input.instruction->print("Generating Taskflow for: ");
+    , to_end_input.instruction->getDescription() input.instruction->print("Generating Taskflow for: ");
```

Comment:
I'm confused by what's happening on this line. Also this freespace_process_manager.cpp file is never built or included anywhere that I can find.

Reply:
This file must have been reintroduced during a rebase. It should be removed.